Frends.Regex.IsMatch/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.0.2] - 2022-08-23
+### Changed
+- Documentation and dependency update.
```

Comment:
Documentation update doesn't have to be recorded. Also please be more specific about what dependency was updated and to what version. Changelog is for users to refer when something has changed, so they have to be specific.